### DISCUSSION TITLE
twin.macro: object null is not iterable

### DISCUSSION THREAD

**Discussion by vutpov** [ben-rogerson/twin.macro#796] [Q&A] [answered]
I'm getting the following error 
![Untitled](https://user-images.githubusercontent.com/42206316/231714286-44bf9483-2031-4e2a-9a2c-6e4d4b5d1cbf.png)

after adding custom theme config to tailwind.config.js.

![Untitled](https://user-images.githubusercontent.com/42206316/231713362-74131afa-c873-420e-8773-9e3f278fffff.png)

Here's the the repo for reproduction. https://github.com/vutpov/test-next

**Comment by ben-rogerson** [comment, marked as answer]
Thanks for providing a repo to test.

The problem is the screen definition - you'll need define the screens as strings (the order of these matter too):

```js
module.exports = {
  theme: {
    screens: {
      xs: "480px",
      sm: "768px",
      md: "1024px",
      lg: "1280px",
      xl: "1440px",
    },
  },
};
```

- [Tailwind screens documentation](https://tailwindcss.com/docs/screens)
